R/create_child_care_db.R

```diff
@@ -78,8 +78,6 @@ child_care_db <- function(root,
 
   env$XWALK_TRACTS <- process.tracts_xwalk(cls = list(raw_pth = raw_pth))
 
-  env$ADJ_TRACTS <- process.adj_tracts(cls = list(raw_pth = raw_pth))
```

Comment:
@holmesjoli is there a reason you removed this? it's causing an error bc ADJ_TRACTS is called to make the XWALK_TRACTS df. I can add it back, just wanted to make sure it wasn't removed for a reason.

Reply:
@sadierichardson this is a mistake! Good catch.